pywatts/core/base_step.py

```diff
@@ -74,10 +74,11 @@ def get_result(self, start: pd.Timestamp, end: Optional[pd.Timestamp], buffer_el
         # Only execute the module if the step is not finished and the results are not yet calculated
         if not self.finished and not (end is not None and self._current_end is not None and end <= self._current_end):
             if not self.buffer or not self._current_end or end > self._current_end:
-                self.cached_result["cached"] = self._compute(start, end)
+                self.cached_result["cached"] = self._compute(start, end, minimum_data)
                 self.cached_result["start"] = start
                 self.cached_result["end"] = end
                 self._current_end = end
+                self.minimum_data = minimum_data
```

Comment:
this is a new argument that is never used, should it be `self.cached_result["minimum_data"]`?